proxy/src/http/prom.rs

```diff
@@ -171,7 +164,7 @@ impl<Q: QueryExecutor + 'static, P: PhysicalPlanner> Proxy<Q, P> {
             .await?;
 
         let cost = begin_instant.saturating_elapsed().as_millis();
-        debug!("Query handler finished, request_id:{request_id}, cost:{cost}ms, query:{query:?}");
+        info!("Handle prom remote query sucess, ctx:{ctx:?}, cost:{cost}ms");
```

Comment:
Will info level lead to too many logs?

Reply:
Will add slow query threshold to reduce log, for now print all query logs